nvflare/job_config/script_runner.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         launch_external_process: bool = False,
         command: str = "python3",
         framework: FrameworkType = FrameworkType.PYTORCH,
+        script_dir: str = None,
```

Comment:
Instead of adding a "script_dir" here in the ScriptRunner, I think it's better to use the:

     job.to_clients(script_dir)

And this way is more consistent how it is used for the server side.

Reply:
But the script_dir / content_dir is not only used for the ScriptRunner. It's a place for all the "contents" needed for the Application. Use the job.to_clients() once can included all the contents needed. The "script_dir" supposed only related to the script itself.

Reply:
My point is once we have the  job.to_clients(script_dir), we don't need the script_dir in the ScriptRunner any more.

Reply:
I am not sure about this. The  job.to_clients(script_dir) function already exist. And it covers all the things needed here for the script_dir. I don't feel necessary for the change any more, from the whole Job perspective.